src/annotator/pdf-sidebar.js

```diff
@@ -12,8 +16,101 @@ const defaultConfig = {
   },
 };
 
+// In accordance with the minimum width PDFJS asserts for its #mainContainer
+// element, we shouldn't try to shrink down the PDF container any narrower
+// than this value when resizing it in side-by-side mode. If the available space
+// left in the viewport after alloting space for the sidebar is less than this
+// value, don't apply side-by-side mode, but instead allow the sidebar to
+// overlap the PDF
+const MIN_PDF_WIDTH = 320;
```

Comment:
The PDF viewer toolbar has a minimum width of ~690px before the layout starts to break, so I would avoid applying side-by-side mode if the space available for the PDF viewer after applying it would be less than say 700px.